docs/guides/migrate/migration-guide.md

```diff
@@ -919,12 +919,12 @@ Unsafe.unsafe { implicit u =>
 
 Similarly, we can do the same for other unsafe operators. Here are some of them:
 
-| ZIO 1.0                        | ZIO 2.x                                                                                                    |
-|--------------------------------|------------------------------------------------------------------------------------------------------------|
-| `runtime.unsafeRunSync(x)`     | `Unsafe.unsafe { implicit u => runtime.unsafe.run(x) }`                                                    |
-| `runtime.unsafeRunTask(x)`     | `Unsafe.unsafe { implicit u => runtime.unsafe.run(x).foldExit(cause => throw cause.squashTrace, identity)` |
-| `runtime.unsafeRunAsync_(x)`   | `Unsafe.unsafe { implicit u => runtime.unsafe.fork(x).unsafe.addObserver(_fold(identity, identity))`       |
-| `runtime.unsafeRunToFuture(x)` | `Unsafe.unsafe { implicit u => runtime.unsafe.runToFuture(x) }`                                            |
+| ZIO 1.0                        | ZIO 2.x                                                                                                |
+|--------------------------------|--------------------------------------------------------------------------------------------------------|
+| `runtime.unsafeRunSync(x)`     | `Unsafe.unsafe { implicit u => runtime.unsafe.run(x) }`                                                |
+| `runtime.unsafeRunTask(x)`     | `Unsafe.unsafe { implicit u => runtime.unsafe.run(x).getOrThrow() }`                                   |
+| `runtime.unsafeRunAsync_(x)`   | `Unsafe.unsafe { implicit u => runtime.unsafe.fork(x).unsafe.addObserver(_fold(identity, identity)) }` |
```

Comment:
Don't think there is any point in adding an observer here if we are not going to do anything.